snapcraft/plugins/jhbuild.py

```diff
@@ -88,7 +88,6 @@
     'make',
     'ninja-build',
     'pkg-config',
-    'pkgconf',
```

Comment:
How come the integration test didn't catch this before?

Reply:
Answering my own q, it seems we didn't run adt before, and indeed in the archive this isn't even a dependency.